pages/phases/assessments/written-assessments.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: article
+permalink: /hiring-phases/assessments/written-assessments/
+section: hiring-phases
+sidenav: hiring-phases
+phase: assessments
+sub-phase-order: 020
+title: Written Assessments
+intro: SMEs may conduct one or two rounds of written assessments in the assessment phase.
+---
+
+<p class="usa-intro">
+  {{ page.intro }}
+</p>
+
+You may choose to do a written assessment for one or both assessment phases.
```

Comment:
i dont like this line bc it assumes no structured interview which i dont love recommending w/o a team conversation. i think we've always said an interview is mandatory part of SMEQA. we can always revisit in the name of flexibility and agencies can do whatever they want.

Reply:
Good call.

Reply:
Edited to this:

If your SMEs opt for a written assessment or demonstration project instead of a first-round interview, you will need to prepare questions and training for each different assessment round.

The assessment type should be appropriate to evaluate the applicant's proficiency level within the examined competencies

In this commit: https://github.com/usds/SME-QA-Website/pull/243/commits/a553fb2e20c5bcea5cb5372200db7875cbddeb80